src/Octoshift/Services/GithubApi.cs

```diff
@@ -1074,25 +1074,23 @@ mutation abortRepositoryMigration(
 
     public virtual async Task<string> UploadArchiveToGithubStorage(string orgDatabaseId, bool isMultipart, string archiveName, Stream archiveContent)
     {
-        using var httpContent = new StreamContent(archiveContent);
+        using var streamContent = new StreamContent(archiveContent);
         string response;
 
         if (isMultipart)
         {
             var url = $"https://uploads.github.com/organizations/{orgDatabaseId.EscapeDataString()}/gei/archive/blobs/uploads";
 
-            using var content = new MultipartFormDataContent
-            {
-                { httpContent, "archive", archiveName }
-            };
+            using var multipartFormDataContent = new MultipartFormDataContent();
+            multipartFormDataContent.Add(streamContent, "archive", archiveName);
```

Comment:
Lets disable `IDE0028` warning as it's a false positive in this case because if there's an exception during the initialization of a using variable it won't get disposed.
```suggestion
#pragma warning disable IDE0028
            using var multipartFormDataContent = new MultipartFormDataContent();
            multipartFormDataContent.Add(streamContent, "archive", archiveName);
#pragma warning restore
```